### ISSUE TITLE
Test failure from release tarball

### ISSUE THREAD

**Issue by orlitzky** [james-d-mitchell/gaplint#30]
When using the tagged release from e.g. https://github.com/james-d-mitchell/gaplint/archive/refs/tags/v1.1.5.tar.gz, one test fails because it's looking for `.git`:

```
$ pytest
============================= test session starts ==============================
platform linux -- Python 3.11.9, pytest-8.1.1, pluggy-1.4.0
rootdir: /home/mjo/src/gaplint-1.1.5
configfile: pyproject.toml
plugins: pkgcore-0.12.27
collected 22 items                                                             

tests/test_gaplint.py ...................F..                             [100%]

=================================== FAILURES ===================================
_____________ TestConfigYAMLFile.test_with_config_file_parent_root _____________

self = <tests.test_gaplint.TestConfigYAMLFile testMethod=test_with_config_file_parent_root>

    def test_with_config_file_parent_root(self):
>       os.rename(".git", ".tmp_git")
E       FileNotFoundError: [Errno 2] No such file or directory: '.git' -> '.tmp_git'

tests/test_gaplint.py:245: FileNotFoundError
=========================== short test summary info ============================
FAILED tests/test_gaplint.py::TestConfigYAMLFile::test_with_config_file_parent_root - FileNotFoundError: [Errno 2] No such file or directory: '.git' -> '.tmp_git'
========================= 1 failed, 21 passed in 0.44s =========================
```

**Comment by james-d-mitchell** [comment]
Thanks @orlitzky, that's not such a good test, will fix asap.

**Comment by james-d-mitchell** [comment]
Resolved in v1.2.0
